### DISCUSSION TITLE
2011/07/13/%E4%B8%AD%E5%9B%BD%E4%BA%BA%E7%9A%84%E8%B5%B7%E6%BA%90

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#338] [Announcements]
# 2011/07/13/%E4%B8%AD%E5%9B%BD%E4%BA%BA%E7%9A%84%E8%B5%B7%E6%BA%90

据说在汉族人特有的身体特征中，最具普遍性的是複型小指甲：一只脚或双脚的小脚趾甲是分成左右两片的。我观察了一下身边的亲人，看他们是否符合这一特征。就我观察的的有限的几个人来说，南方（江南）人複型小指甲比较明显，北方人不明显。我自己身上南北方的基因各占一半：我记得我小时候是有複型小指甲的，可是今天举起脚丫来一看，竟然没了。老婆家祖上好几代都生活在山东，她就没有複型小指甲。豆豆也没有，豆豆身上还是北方基因占主的。

https://blog.qizhen.xyz/2011/07/13/%E4%B8%AD%E5%9B%BD%E4%BA%BA%E7%9A%84%E8%B5%B7%E6%BA%90

**Comment by ruanqizhen** [comment]
之前博客下的留言：

革之
2011年07月14日 05:31 编辑
非洲起源论有历史了，从最近考古看，已经站不住脚了
还有，我们村所有本地人的小脚趾是复指甲，地处长江边上，古汉人的一支

回复
ruanqizhen
2011年07月14日 09:24 编辑
不知道最近考古考到了啥。
但不管怎样，我觉得考古证据都不如基因证据可靠。因为所谓“考古证据”主观性太强，比人一个猿人头骨，有人觉得他更像欧洲人，有人会觉得他更像亚洲人。基因的不同却可以比较客观的量化：比如，有3个基因不同的就是比2个基因不同的差距大。

回复
Adeline
2011年07月13日 22:03 编辑
以前还看过一个说法，大部分汉族人都有蒙古族血统。蒙古人的特征和汉族人的特征几乎难分你我，比如，眼睛细长。果真这样吗？

回复
ruanqizhen
2011年07月14日 09:22 编辑
从大的角度分，汉族和蒙古族以及整个东北亚的其他民族都属于蒙古人种。“蒙古人种”只是个名字，这个人种中肯定是汉人占主。估计叫蒙古人种是因为他们共同起源与两万年前的蒙古大草原。
蒙古人种共有的特征很多的，包括：婴儿屁股上有青斑胎记；上门牙不是平的呈铲子型；内眦褶等等；耳屎是干的等等。

回复
Emily
2011年07月13日 21:32 编辑
小时候木有複型小指甲，长大才有的~~不过貌似我家那块不南不北~~

回复
